### PR TITLE
Adding GetContainerLogs method to GeoMasterProvider

### PR DESCRIPTION
**Pull request by puneetg1983**

### CODE REVIEW

File: src/Diagnostics.DataProviders/DataProviderLogDecorator.cs [hunk 1]
```diff
@@ -184,6 +184,11 @@ public Task<T> MakeHttpGetRequestWithFullPath<T>(string fullPath, string querySt
             return MakeDependencyCall(null, _geomasterDataProvider.MakeHttpGetRequestWithFullPath<T>(fullPath, queryString, apiVersion));
         }
 
+        public Task<string> GetContainerLogs(string subscriptionId, string resourceGroupName, string name)
```
Comment (ShekharGupta1988) [inline src/Diagnostics.DataProviders/DataProviderLogDecorator.cs]: I am guessing its linux container (docker or ACR).  Remember there is support for Windows Containers now also in antares. 
If this method is only targeting Linux, maybe call it 'GetLinuxContainerLogs' ?

File: src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs [hunk 2]
```diff
@@ -364,6 +364,36 @@ public async Task<T> MakeHttpGetRequestWithFullPath<T>(string fullPath, string q
...
+        ///     var name = cxt.Resource.Name;
+        ///     
+        ///     string containerLogs = await dp.GeoMaster.GetContainerLogs(subId, rg, name);
+        ///         
+        ///     // do any processing on the string variable containerLogs
+        /// }
+        /// </code>
+        /// </example>
+        /// <returns></returns>
+        public async Task<string> GetContainerLogs(string subscriptionId, string resourceGroupName, string name)
```
Comment (ShekharGupta1988) [inline src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs]: (This is probably true of all methods).
Recently, @hforeste did a change to runtime to separate out site name with slot name and pass it in Context resource.

Does this method "SitePathUtility.GetSitePath()" take into consideration slotName also? or parsing 'app(slot)' perhaps

File: src/Diagnostics.DataProviders/DataProviderLogDecorator.cs (hunk 1 above)
Comment (puneetg1983) [reply src/Diagnostics.DataProviders/DataProviderLogDecorator.cs]: good point, changed it to 'GetLinuxContainerLogs'

File: src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs (hunk 2 above)
Comment (puneetg1983) [reply src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs]: Yup it does. It allows passing site+slot as siteName(slot)

**Pull request merged**